code/modules/power/tesla/energy_ball.dm

```diff
@@ -23,7 +23,11 @@
 	appearance_flags = LONG_GLIDE
 	density = TRUE
 	plane = MASSIVE_OBJ_PLANE
-	light_range = 6
+	light_system = MOVABLE_LIGHT
+	light_range = 10
+	light_power = 10
```

Comment:
This is too big for the movable lighting system, it has a limit of ~5-6 before it getes weird because of the way it works